digdag-tests/src/test/java/acceptance/SchedulerIT.java

```diff
@@ -42,4 +44,30 @@ public void testScheduler()
         }
         assertTrue(false);
     }
+
+    @Test
+    public void testInvalidScheduler()
+            throws Exception
+    {
+        try {
+            copyResource("acceptance/scheduler/invalid.dig", root().resolve("test.dig"));
+            main("scheduler", "--project", root().toString());
+            fail();
+        } catch (Throwable ie) {
+            assertThat(ie.getCause().getCause().toString(), Matchers.containsString("Parameter 'skip_on_over_time' is not used at schedule. > Did you mean '[skip_on_overtime]'"));
+        }
+    }
+
+    @Test
+    public void testInvalidOpScheduler()
+            throws Exception
+    {
+        try {
+            copyResource("acceptance/scheduler/invalid_op.dig", root().resolve("test.dig"));
+            main("scheduler", "--project", root().toString());
+            fail();
+        } catch (Throwable ie) {
+            assertThat(ie.getCause().getCause().toString(), Matchers.containsString("Parameter 'wrong>' is not used at schedule."));
+        }
+    }
```

Comment:
can you add some more tests?:
* A test that verifies REST API response code. I think API response code should be 422. It's very bad if it's 5xx. If API response code is unexpected, additional code will be necessary around ProjectResource.validateWorkflowAndSchedule.
* what if unknown scheduler type is set (like `no_such_scheduler>: abc`)

Reply:
you can copy testing mechanism from ServerErrorStatusCodeIT to this SchedulerIT.

Reply:
I added tests for them here. 
https://github.com/treasure-data/digdag/pull/1381/commits/cca6e1f7ee3f7507b0948448ae5212676acf0e19